statannot/statannot.py

```diff
@@ -78,6 +81,17 @@ def stat_test(
             u_stat,
             pval,
         )
+    elif test == 'anova':
+        u_stat, pval = stats.f_oneway(
```

Comment:
For clarity in the code, and accurate reporting below, I'd change `u_stat` to `f_stat`.